api/wikis/serializers.py

```diff
@@ -114,6 +158,12 @@ class RegistrationWikiSerializer(WikiSerializer):
         filter={'target': '<_id>'}
     )
 
+    def update(self, instance, validated_data):
+        raise MethodNotAllowed(method=self.context['request'].method)
+
+    def create(self, validated_data):
+        raise MethodNotAllowed(method=self.context['request'].method)
```

Comment:
Will these methods get called or is this caught earlier?

Reply:
I believe the `create` override is needed but please double check my reasoning:

`RegistrationWikiSerializer` is used in the `RegistrationWikiList` view, which inherits from the `NodeWikiList` view. The `NodeWikiList` view doesn't raise `MethodNotAllowed` for registrations like views with the `WikiMixin` do, therefore the error will be raised here.

The `update` override is now obsolete; will remove.